next/modules/cards-and-rows/EventCard.tsx

```diff
@@ -40,12 +41,13 @@ const EventCard = ({
 
   return (
     <div className="relative">
-      <img
-        className="h-[200px] w-full flex-1 object-cover"
-        alt={t('eventDetailImagePlaceholder')}
-        src={listingImage?.url || coverImage?.url || Placeholder.src}
-        height="200px"
-      />
+      <div className="relative h-[163px] w-[271px] object-cover">
+        <Image
+          fill
+          alt={t('eventDetailImagePlaceholder')}
+          src={listingImage?.url || coverImage?.url || Placeholder.src}
+        />
+      </div>
```

Comment:
Never use fixed width for cards, they must be responsive.

`object-cover` is usually used on Image together with `fill`:
```
<Image fill className="object-cover" .../>
```

Try to use tailwind's `aspect-ratio` class.